atqa-framework/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "atqa-waves",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "eslint-check": "eslint . --ext .ts --ext .js ; gherkin-lint",
+    "eslint-fix": "eslint . --ext .ts --ext .js --fix",
+    "lint:typescript": "tsc --noEmit",
+    "report:open": "allure serve output",
+    "report:remove": "rm -rf output/*",
+    "remove:extension": "rm -rf extension/*",
+    "build:extension": "cd .. && npm run dist",
+    "init:folders": "mkdir buffer && mkdir extension",
+    "copy:buffer:extension": "cp -Rv ./buffer/chrome ./buffer/opera ./buffer/edge ./extension/",
+    "e2e:test": "yarn report:remove ; codeceptjs run --features --steps --grep '^(?!.*@ignore)'",
+    "e2e:test:opera": "yarn e2e:test --grep '^(?!.*@ignore)(?=.*@opera)'",
+    "e2e:test:edge": "yarn e2e:test --grep '^(?!.*@ignore)(?=.*@edge)'",
+    "e2e:test:chrome": "yarn e2e:test --grep '^(?!.*@ignore)(?=.*@chrome)'",
+    "e2e:test:debug": "yarn report:remove ; codeceptjs run --grep '^(?!.*@ignore)(?=.*@debug)' --debug --verbose --debug-brk ; yarn report:open"
+  },
+  "dependencies": {
+    "@codeceptjs/helper": "^2.0.1",
+    "@types/fs-extra": "^9.0.13",
+    "@wdio/types": "^7.20.3",
+    "codeceptjs": "^3.3.4",
+    "dayjs": "^1.11.4",
+    "dotenv-safe": "^8.2.0",
+    "firefox-profile": "^4.2.2",
+    "fs-extra": "^10.1.0",
+    "http-status": "^1.5.2",
+    "ts-node": "^10.8.2",
+    "typescript": "^4.7.4",
+    "webdriverio": "^7.20.7",
+    "@faker-js/faker": "^7.4.0",
+    "expect": "^26.6.2"
+  },
+  "devDependencies": {
+    "@wdio/firefox-profile-service": "^5.21.0",
+    "@wdio/selenium-standalone-service": "^7.20.7",
+    "eslint": "^7.14.0",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "eslint-config-prettier": "^7.1.0",
+    "eslint-plugin-codeceptjs": "^1.3.0",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-prettier": "^3.3.0",
+    "@typescript-eslint/eslint-plugin": "^4.9.1",
+    "@typescript-eslint/parser": "^4.9.1",
+    "prettier-eslint": "^12.0.0",
+    "gherkin-lint": "4.2.2"
+  },
+  "prettier": {
+    "singleQuote": true
+  }
```

Comment:
May be use root `.prettierrc.json`?

Reply:
no, if it is possible to specify all in one it should be like that